alibi_detect/saving/tests/test_saving.py

```diff
@@ -259,6 +268,22 @@ def preprocess_hiddenoutput(classifier, backend):
     return preprocess_fn
 
 
+@parametrize('cfg', CFGS)
+def test_load_simple_config(cfg, tmp_path):
+    save_dir = tmp_path
+    x_ref_path = str(save_dir.joinpath('x_ref.npy'))
+    cfg_path = save_dir.joinpath('config.toml')
+    # Save x_ref in config.toml
+    x_ref = cfg['x_ref']
+    np.save(x_ref_path, x_ref)
+    cfg['x_ref'] = 'x_ref.npy'
+    # Save config.toml then load it
+    with open(cfg_path, 'w') as f:
+        toml.dump(cfg, f)
+    cd = load_detector(cfg_path)
```

Comment:
Is it necessary to dump to `.toml` first instead of calling `load_detector` on the dictionary? I suppose if we're strictly testing loading of `.toml` then it's ok.

Reply:
I agree with you on this one, being able to pass config dicts to `load_detector` is something I think is nice. However, I can't recall why now but sometime back we decided to remove this capability (I think just to reduce complexity). The function signature for `load_detector` is currently ` load_detector(filepath: Union[str, os.PathLike], **kwargs)`.

I'd quite like to revisit this at some point...